doc/how-to/getting_started_go.rst

```diff
@@ -1,256 +1,279 @@
 .. _getting_started-go:
 
---------------------------------------------------------------------------------
 Connecting from Go
---------------------------------------------------------------------------------
+==================
 
-.. _getting_started-go-pre-requisites:
+**Examples on GitHub**: `sample_db <https://github.com/tarantool/doc/tree/latest/doc/code_snippets/snippets/connectors/instances.enabled/sample_db>`_, `go <https://github.com/tarantool/doc/tree/latest/doc/code_snippets/snippets/connectors/go>`_
 
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-Pre-requisites
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The tutorial shows how to use the 2.x version of the `go-tarantool <https://github.com/tarantool/go-tarantool>`__ library to create a Go application that connects to a remote Tarantool instance, performs CRUD operations, and executes a stored procedure.
```

Comment:
I'd use the direct order of words:
```
... how to use the go-tarantool 2.x library
```
because this page is firstly about go-tarantool, and secondarily - about 2.x version.
Besides, if I get it right, we're not gonna provide content on earlier go-tarantool versions, so no sense in highlighting the version in the first place.